polaris.shopify.com/src/components/Longform/Longform.module.scss

```diff
@@ -161,6 +167,14 @@
     font-size: var(--font-size-200);
     border-collapse: collapse;
 
+    p {
```

Comment:
Ensures paragraphs rendered in tables don't have longform content level spacing.